src/data/repositories/UserD2ApiRepository.ts

```diff
@@ -199,6 +199,24 @@ export class UserD2ApiRepository implements UserRepository {
         return userData$.map(({ objects }) => objects);
     }
 
+    public listAll(
+        options: ListOptions,
+        state: { initialPage: number; users: User[] } = { initialPage: 1, users: [] }
+    ): FutureData<User[]> {
+        const { initialPage, users } = state;
+        return this.list({ ...options, page: initialPage }).flatMap(({ pager, objects }) => {
```

Comment:
Can we increase the pageSize to 100 by default? It's not a big difference from 50 and it'll generate less requests.